pkg/reconciler/v1alpha1/revision/config/network.go

```diff
@@ -71,6 +71,7 @@ func NewNetworkFromConfigMap(configMap *corev1.ConfigMap) (*Network, error) {
 	nc := &Network{}
 	if ipr, ok := configMap.Data[IstioOutboundIPRangesKey]; !ok {
 		// It is OK for this to be absent, we will elide the annotation.
+		nc.IstioOutboundIPRanges = "*"
```

Comment:
@mdemirhan Does this make sense?

Reply:
cc @tcnghia

Reply:
Looks correct to me.

Reply:
* and empty have different meanings. From the config map:
```
  # If omitted or set to "", value of global.proxy.includeIPRanges
  # provided at Istio deployment time is used. In default Knative serving
  # deployment, global.proxy.includeIPRanges value is set to "*".
```

* will intercept everything. Empty will default to Istio's installation defaults, which is vendor specific.

I think it is ok to change this behavior, given that we will soon remove this entire thing and replace is with proper Egress rules from Istio (hopefully the change will be in soon to enable us that). But for clarity, we should change the documentation in the config map and mention that both *, omitted or "" behaves the same way.